ecocode-rules-specifications/src/main/rules/EC4/EC4.json

```diff
@@ -1,15 +0,0 @@
-{
```

Comment:
I don't agree to delete all EC4 rule because not deprecated and still in usage (check "EC4" rule in array above) for PHP and Python

Reply:
ok, reverted